classes/ActionScheduler_Store.php

```diff
@@ -14,6 +14,26 @@
 	/** @var ActionScheduler_Store */
 	private static $store = NULL;
 
+	/**
+	 * Fields that can be stored with actions.
+	 *
+	 * @var array
+	 */
+	protected $action_fields = array(
+		'action_id'            => 1,
+		'hook'                 => 1,
+		'status'               => 1,
+		'scheduled_date_gmt'   => 1,
+		'scheduled_date_local' => 1,
+		'args'                 => 1,
+		'schedule'             => 1,
+		'group_id'             => 1,
+		'attempts'             => 1,
```

Comment:
There is no concept of `attempts` in Action Scheduler core, only in the custom tables plugin. The number of attemps is stored in the `menu_order` column, but it's never used. It's handy to be able to use/update, but given it's not implemented in `ActionScheduler_wpPostStore` (because it's not set in `ActionScheduler_wpPostStore::$field_map`), I'm not sure whether it should be included.

Reply:
> The number of attemps is stored in the `menu_order` column, but it's never used. 

I didn't realize that was the case. Good to know. I was assuming I would just have to ignore the number of attempts for the post data store. I could certainly include that functionality in the post data store.